package.json

```diff
@@ -33,15 +33,16 @@
 	"scripts": {
 		"bump": "bump package.json package-lock.json",
 		"check": "npm run check-rules && npm run lint",
-		"check-js": "eslint-find-rules -nd js/index.js",
+		"check-js": "eslint-find-rules -nd js/index.js && eslint-config-prettier js/index.js",
```

Comment:
Just curious: what type of check is performed by `eslint-config-prettier`?

Reply:
It checks if your configuration contains any rules that are unnecessary or conflict with Prettier